packages/ckeditor5-find-and-replace/src/findandreplaceediting.js

```diff
@@ -16,8 +16,7 @@ import FindNextCommand from './findnextcommand';
 import FindPreviousCommand from './findpreviouscommand';
 import FindAndReplaceState from './findandreplacestate';
 
-// eslint-disable-next-line ckeditor5-rules/ckeditor-imports
-import { scrollViewportToShowTarget } from '@ckeditor/ckeditor5-utils/src/dom/scroll';
+import { scrollViewportToShowTarget } from 'ckeditor5/src/utils';
```

Comment:
This is an unrelated change to the scope of the PR. I believe it should land on `#master`. Could you report a follow-up for it?

Reply:
I believe I saw an error because of that, so I fixed it. I don't see it anymore, so I'm going to remove this change from PR